src/sidebar/components/test/annotation-test.js

```diff
@@ -116,10 +131,11 @@ describe('annotation', function() {
     before(function() {
       angular.module('h', [])
         .component('annotation', annotationComponent())
+        .component('annotationActionButton', {
+          bindings: require('../annotation-action-button').bindings,
+        })
         .component('markdown', {
-          bindings: {
-            customTextClass: '<?',
-          },
+          bindings: require('../markdown').bindings,
```

Comment:
Do I understand correctly that this just creates "empty" annotation action button and markdown components for the tests? i.e. a bug in `annotation-action-button.{html,js}` won't cause `annotation-test.js` to fail?